wiki/Tournaments/VTSS/1/en.md

```diff
@@ -0,0 +1,1150 @@
+
+##### Winner's bracket
+
+1. All players start in the winner's bracket from Round of 128 onwards until they lose a set, sending them down to the loser's bracket.
+2. Only during Grand Finals if the player from the winner's bracket loses, then there will a set reset.
+
+##### Loser's bracket
+
+1. After the first week of matches, the losers of Round of 128 will go against each other in the Round of 64 loser's bracket.
+2. In Round of 32 onwards, players who start the round in the loser's bracket will play two matches per round.
+3. The first match would be the player that won their last loser's match round against a player that lost their winner's bracket match last round.
+4. The second match of the round would be two loser's bracket players that won their last round.
+5. This will continue until Grand Finals where if the loser's bracket player win, there will be a set reset.
+
+### Win conditions and map bans
+
+1. Round of 128 and Round of 64: Best-of-7 (first to 4 points) and 1 ban.
+2. Round of 32: Best-of-9 (first to 5 points) and 1 ban.
+3. Round of 16: Best-of-9 (first to 5 points) and 2 bans.
+4. Quarterfinals and Semifinals: Best-of-11 (first to 6 points) and 2 bans.
+5. Finals and Grand Finals: Best-of-13 (first to 7 points) and 2 bans.
+
+### Match procedures
+
+#### Qualifiers
+
+1. Players sign up for time slots to play in qualifiers lobbies.
+2. Players may switch their qualifiers times whenever they want.
+3. Players join the match and play the maps in the order:
+   1. NM1
+   2. NM2
+   3. NM3
+   4. NM4
+   5. HD1
+   6. HD2
+   7. HR1
+   8. HR2
+   9. DT1
+   10. DT2
+
+#### Bracket
+
+1. Players will automatically be filled into matches with the time being based on the time zones players in.
+2. Loser's bracket players after Round of 128 will have 3 matches scheduled for them. The two matches scheduled are potential matches assuming themselves and the respective player win their previous matches.
+3. Both players may agree to reschedule their match up to a week from their original match date. They must post their agreement on the `#reschedules` text channel in the Discord server along with the respective format for rescheduling.
+4. Loser's bracket players can also reschedule their potential loser's bracket matches as well.
+5. Rescheduling is recommended to be done up to **1 day** before the first match from their respective stage starts in order to ensure referees, commentators, and streamers know when the match will occur so they can perform their respective duties.
+6. You will get a notification from your referee that your match is ready 10 - 15 minutes before the schedule. Once you are ready please let them know and they will invite you to the lobby.
+7. If the referee assigned for the match isn't available or they don't show, the match will be assigned to another referee. This will postpone the match by 15 minutes, and if there's still no referee by the given time, the match will be postponed.
```

Comment:
"This will postpone the match ..., and if ..., the match will be postponed" -- sounds weird as well, replace second "postponed" with "cancelled/rescheduled"?

```suggestion
7. If the referee assigned for the match isn't available or they don't show up, the match will be assigned to another referee. This will postpone the match by 15 minutes, and if there's still no referee by the given time, the match will be postponed.
```